src/components/Section/Citizenship/Multiple/CitizenshipItem.jsx

```diff
@@ -116,7 +116,9 @@ export default class CitizenshipItem extends ValidationElement {
             name="Dates"
             {...this.props.Dates}
             className="citizenship-dates"
-            minDateEqualTo
+            maxDate={(this.props.minDate || {}).date}
+            maxDateEqualTo
+            minDateEqualTo={true}
```

Comment:
I think for consistency's sake we should either omit the `={true}` from the places it's used or use it everywhere

Reply:
Yep, agreed!